gnovm/stdlibs/std/coins.gno

```diff
@@ -13,21 +15,77 @@ type Coin struct {
 	Amount int64  `json:"amount"`
 }
 
+// NewCoin returns a new coin with a denomination and amount
+func NewCoin(denom string, amount int64) Coin {
+	return Coin{
+		Denom:  denom,
+		Amount: amount,
+	}
+}
+
+// String provides a human-readable representation of a coin
 func (c Coin) String() string {
+	if c.IsZero() {
+		return ""
+	}
```

Comment:
shouldn't the output be `0DENOM`? I see misleading outputs if we use this method to show info to the user (example: `your current bank account status:`  instead of `your current bank account status: 0DENOM`)

Reply:
Here the issue is the following

if you use

Reply:
Fair point